src/promptflow/tests/test_configs/notebooks/dummy.ipynb

```diff
@@ -74,6 +74,28 @@
     "# create run with default variant\n",
     "base_run = pf.run(flow=flow, data=data, stream=True)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Load flow as a function\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Sync function\n",
```

Comment:
the comment here should be aysnc?

Reply:
OK, let me refine this wording.
load sync flow as async function.